README.md

```diff
@@ -101,3 +97,24 @@ Then run the application (here on Android):
 ```sh
 npm run cordova-android
 ```
+
+## How to build the Chrome extension ``hoverpad.crx`` file?
+
+Assuming the ``chromium-browser`` command is available in your ``PATH``:
+
+```sh
+npm run chrome-web-ext
+```
+
+## How to build the Chrome Store ``hoverpad.zip`` file?
+
+```sh
+npm run chrome-store-zip
+```
+
+## How to build the Firefox extension ``hoverpad.xpi`` file?
+
+```sh
+npm run fx-web-ext
+ls hoverpad/web-ext-artifacts/hoverpad-*.zip
```

Comment:
Maybe I should remove hoverpad and just keep web-ext-artifacts which is built automatically from the fx-web-ext command.